Chromium/Makefile

```diff
@@ -1,9 +1,6 @@
 all: tables guide content dist
 
-# Do not remove exporting of SHARED variable!!! (since
-# shared/transforms/combineremediations.py helper
-# requires its value to be available)
-export SHARED = ../shared
```

Comment:
Here is what happens in the build: `xsltproc --stringparam SHARED_RP "$(realpath $(SHARED))" ...`. We use the variable but we don't need it exported.

At least that's my understanding of it.